crates/vm/levm/src/opcode_handlers/system.rs

```diff
@@ -37,8 +36,13 @@ impl VM {
             .try_into()
             .map_err(|_| VMError::VeryLargeNumber)?;
 
-        let current_memory_size = current_call_frame.memory.len();
+        // OOG VALIDATIONS
```

Comment:
I think this comment isn't correct
```suggestion
        // STATIC CONTEXT VALIDATIONS
```

Reply:
https://github.com/lambdaclass/ethrex/pull/1492/commits/ad9a0c39a623d105e7b327234eb965c38de86260
I wanted a generic comment for out of gas validations.
I ended up changing it for `Validations` only.